recipe/meta.yaml

```diff
@@ -1,47 +1,44 @@
 {% set name = "spyder-kernels" %}
 {% set version = "2.4.0" %}
-{% set hash = "3bd32fbe30a150ab9ccedbd5beb6ddd38bde1d9a8778da52f7edf420bbfb8e51" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-kernels-{{ version }}.tar.gz
+  sha256: 3bd32fbe30a150ab9ccedbd5beb6ddd38bde1d9a8778da52f7edf420bbfb8e51
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
+  noarch: python
+  string: "unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"  # [unix]
+  string: "win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"   # [win]
 
 requirements:
-  build:
-    - python                                # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-
   host:
+    - python >=3.0
     - pip
-    - python
     - setuptools
-
   run:
-    - python
+    - python >=3.0
     - cloudpickle
     - ipykernel >=6.16.1,<7.0.0
     - ipython >=7.31.1,<8.0.0
-    - jupyter_client >=7.3.1,<8.0.0  # [python_impl == 'pypy']
-    - jupyter_client >=7.3.4,<8.0.0  # [python_impl == 'cpython']
+    - jupyter_client >=7.3.4,<8.0.0
     - pyzmq >=22.1.0
     - wurlitzer >=1.0.3  # [not win]
```

Comment:
Is this still not needed on Windows? Because it is noarch in conda-forge: https://github.com/conda-forge/wurlitzer-feedstock

Reply:
The thing is `wurlitzer` is not really necessary on Windows because it can only do its magic only on Posix systems. That's why we don't require it there.

However, we have a method that prevents from loading the `wurlitzer` IPython extension on Windows:

https://github.com/spyder-ide/spyder-kernels/blob/0c9bc6edc2a04c9867e7ffc8fd103e30534b6842/spyder_kernels/console/kernel.py#L849-L860

So I guess it's ok to have a `noarch` package here? I think @mattip (for PyPy) and other people that want to have a Spyder-kernel in more esoteric platforms would be happy with this change, right?

Reply:
Would make it slightly more easy to maintain but doesn't really matter from that perspective, just my 2 cents. :)

Reply:
Just to clarify, the package is already noarch with this PR, the question is only if we can reduce it to one noarch package or need 2, the former would make maintenance slightly easier but it's no big deal.

Reply:
Two is already 9x better than the current 18 (for 2.4.0)